go.mod

```diff
@@ -69,11 +70,24 @@ require (
 )
 
 require (
+	cloud.google.com/go v0.110.8 // indirect
+	cloud.google.com/go/compute v1.23.1 // indirect
+	cloud.google.com/go/compute/metadata v0.2.3 // indirect
+	cloud.google.com/go/iam v1.1.3 // indirect
+	cloud.google.com/go/storage v1.30.1 // indirect
+	cosmossdk.io/client/v2 v2.0.0-beta.1 // indirect
+	cosmossdk.io/x/circuit v0.1.0 // indirect
+	cosmossdk.io/x/evidence v0.1.0 // indirect
+	cosmossdk.io/x/feegrant v0.1.0 // indirect
+	cosmossdk.io/x/nft v0.1.0 // indirect
+	cosmossdk.io/x/upgrade v0.1.1 // indirect
```

Comment:
Careful here if there are any dependencies in these modules on the changes you guys make down the road it's important to discern what's specifically being imported from mainline sdk.

Reply:
Yeah we would appropriately replace the dependencies when such case arises.